ClimsoftVer4/ClimsoftVer4/frmDataForms.vb

```diff
@@ -1,100 +1,69 @@
 ﻿Public Class frmDataForms
-
-    Dim da As MySql.Data.MySqlClient.MySqlDataAdapter
-    Dim ds As New DataSet
-    Dim sql As String
-    Dim conn As New MySql.Data.MySqlClient.MySqlConnection
-    Dim MyConnectionString As String
-    Dim cmd As New MySql.Data.MySqlClient.MySqlCommand
+    Dim dataCall As New DataCall
+    Dim dataTable As DataTable
 
     Private Sub frmDataForms_Load(sender As Object, e As EventArgs) Handles Me.Load
-        Dim col(2) As String
-        Dim itm As New ListViewItem
-
-        MyConnectionString = frmLogin.txtusrpwd.Text
-        conn.ConnectionString = MyConnectionString
+        'Dim sql As String
+        'Dim conn As New MySql.Data.MySqlClient.MySqlConnection
+        'Dim MyConnectionString As String
+        'Dim cmd As New MySql.Data.MySqlClient.MySqlCommand
+        'MyConnectionString = frmLogin.txtusrpwd.Text
+        'conn.ConnectionString = MyConnectionString
 
         ' Add a record for key entry mode if not exists
-        Try
-            Dim qry As MySql.Data.MySqlClient.MySqlCommand
-            sql = "ALTER TABLE `data_forms` ADD COLUMN `entry_mode` TINYINT(2) NOT NULL DEFAULT '0' AFTER `sequencer`;"
-            conn.Open()
-            qry = New MySql.Data.MySqlClient.MySqlCommand(sql, conn)
-            qry.CommandTimeout = 0
-            qry.ExecuteNonQuery()
+        'Try
+        '    Dim qry As MySql.Data.MySqlClient.MySqlCommand
+        '    sql = "ALTER TABLE `data_forms` ADD COLUMN `entry_mode` TINYINT(2) NOT NULL DEFAULT '0' AFTER `sequencer`;"
+        '    conn.Open()
+        '    qry = New MySql.Data.MySqlClient.MySqlCommand(sql, conn)
+        '    qry.CommandTimeout = 0
+        '    qry.ExecuteNonQuery()
+
+        '    conn.Close()
+        'Catch ex As Exception
```

Comment:
@smachua I have commented out this block because I still don't understand why the form always alters the table on load? Was this a temporary measure? I can put this back if it's still needed? If not I can delete the comment.